paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/wallet/SavedPaymentMethodsActivity.kt

```diff
@@ -24,16 +24,31 @@ import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import com.stripe.android.ExperimentalSavedPaymentMethodsApi
 import com.stripe.android.paymentsheet.example.samples.ui.shared.PaymentSheetExampleTheme
+import com.stripe.android.paymentsheet.wallet.SavedPaymentMethodsSheet
 
+@OptIn(ExperimentalSavedPaymentMethodsApi::class)
 internal class SavedPaymentMethodsActivity : AppCompatActivity() {
     private val viewModel by viewModels<SavedPaymentMethodsViewModel>()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        val savedPaymentMethodsSheet = SavedPaymentMethodsSheet.create(
+            activity = this,
+            configuration = SavedPaymentMethodsSheet.Configuration(
```

Comment:
Should we make Configuration use a builder pattern instead? We've talked about doing something similar for PaymentSheet.

Reply:
What is the benefit of a builder here?

Reply:
Mostly just making it easier to add new configuration options in the future.

We should definitely punt this decision from this PR. But let's talk about it offline, before we submit for API review.